api/src/main/java/org/confeti/controllers/dto/core/ReportResponse.java

```diff
@@ -11,7 +11,7 @@
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
-public class TagResponse {
+public class ReportResponse {
     private String conferenceName;
     private Map<Integer, Map<String, Long>> years;
```

Comment:
Maybe we should call it `statsYears` or just `stats`?

Reply:
I think years is clear, because it statistics for each year